pepdbagent/modules/namespace.py

```diff
@@ -204,3 +212,54 @@ def info(self, limit: int = DEFAULT_LIMIT_INFO) -> ListOfNamespaceInfo:
             limit=limit,
             results=list_of_results,
         )
+
+    def stats(self, namespace: str = None, monthly: bool = False) -> NamespaceStats:
+        """
+        Get statistics for project in the namespace or for all projects in the database.
+
+        :param namespace: namespace name [Default: None (all projects)]
+        :param monthly: if True, get statistics for the last 3 years monthly, else for the last 3 months daily.
+        """
+        if monthly:
+            number_of_month = 3
+        else:
+            number_of_month = 12 * 3
+        today_date = datetime.today().date() + timedelta(days=1)
+        three_month_ago = today_date - timedelta(days=number_of_month * 30 + 1)
+        statement_update = select(Projects.last_update_date).filter(
```

Comment:
rename to `statement_last_updated` and `statement_created_date` to be more explicit?